sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/MetricsData.cs

```diff
@@ -33,5 +35,14 @@ public MetricsData(int version, Metric metric, MetricPoint metricPoint) : base(v
                 }
             }
         }
+
+        public MetricsData(int version) : base(version)
```

Comment:
This can be a function on `ResourceExtensions`. That way we don't have to add this ctor

Reply:
As we have an implementation and to follow the design principles, I kept here.